src/interface.ts

```diff
@@ -98,6 +98,12 @@ export interface CalculatorCommonProps {
    * Text change event.
    */
   onTextChange?: (text: string) => void
+
+
+  /**
```

Comment:
This comment causing `[tslint] asterisks in jsdoc must be aligned [jsdoc-format]` error raised. Please do :
```
/**
   * Digit align display
   */
```
instead and remove extra line above doc comment.